index.bs

```diff
@@ -841,16 +841,35 @@ The [=effective origin=] of an {{XRSpace}} can only be observed in the coordinat
   1. Let |transform| be |pose|'s {{XRPose/transform}}.
-  1. Set |transform|'s {{XRRigidTransform/position}} to the location of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
-  1. Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+  1. Query the [=/XR device=]'s tracking system for |space|'s pose relative to |baseSpace| at the time represented by |frame|, then perform the following steps:
+    <dl class="switch">
+      <dt> If the tracking system provides a [=6DoF=] pose:
+        <dd> Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |transform|'s {{XRRigidTransform/position}} to the position of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |pose|'s {{XRPose/emulatedPosition}} to <code>false</code>.
+
+      <dt> Else if the tracking system provides a [=3DoF=] pose or indicates that the pose position is emulated:
+        <dd> Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |transform|'s {{XRRigidTransform/position}} to the tracking system's best estimate of the position of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=]. This MAY use a computed offset such as a neck or arm model. If a position estimate is not available, the last known position MUST be used.
+        <dd> Set |pose|'s {{XRPose/emulatedPosition}} to <code>true</code>.
+
+      <dt> Else if |space|'s pose relative to |baseSpace| has been determined in the past:
+        <dd> Set |transform|'s {{XRRigidTransform/position}} to the last known position of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |transform|'s {{XRRigidTransform/orientation}} to the last known orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |pose|'s {{XRPose/emulatedPosition}} boolean to <code>true</code>.
+      
+      <dt> Else if |space|'s pose relative to |baseSpace| has never been determined:
+        <dd> Set |pose| to <code>null</code>.
+    </dl>
+
+NOTE: The {{XRPose}}'s {{XRPose/emulatedPosition}} boolean does not indicate whether |baseSpace|'s position is emulated or not, only whether evaluating |space|'s position relative to |baseSpace| relies on emulation. For example, a controller with [=3DoF=] tracking would report poses with an {{XRPose/emulatedPosition}} of <code>true</code> when its {{targetRaySpace}} or {{gripSpace}} are queried against an {{XRReferenceSpace}}, but would likely report an {{XRPose/emulatedPosition}} of <code>false</code> if the pose of the {{targetRaySpace}} was queried in {{gripSpace}}, because the relationship between those two spaces should be known exactly.
```

Comment:
This seems to be at odds with the 3DOF pose thing? i.e. imagine the device has a 3DOF input controller `getPose(grip, targetRay)` should not return an emulated pose as explained here, but the spec text indicates that all 3DOF poses are emulated=true

Reply:
I clarified the text for the "6DoF pose" branch and the "3DoF pose" branch to make clear that these are reasoning about whether that exact relation of `space` to `baseSpace` is 6DoF or 3DoF at the moment, and not whether `space` itself is in some 6DoF or 3DoF state on its own:

> * If the tracking system provides an actively tracked or statically known [=6DoF=] pose for `space`'s pose relative to `baseSpace`:

> * Else if the tracking system provides a [=3DoF=] pose or a [=6DoF=] pose whose position is estimated for `space`'s pose relative to `baseSpace`:

In the case of a 3DoF controller that always estimates its position, `getPose(gripSpace, targetRaySpace)` would now more clearly fall into the first bullet, and so `emulatedPosition` would be `false`.

Reply:
Looks good, thanks!